python/ray/experimental/multiprocessing/pool.py

```diff
@@ -106,6 +106,12 @@ def next_ready_index(self, timeout=None):
 
 
 class AsyncResult(object):
+    """A class that provides the interface to results from calls to
```

Comment:
BTW, "The summary line for a class docstring should fit on one line." 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html